Chat App/Chat App/ChatTableView/MessageTableViewCell.swift

```diff
@@ -0,0 +1,206 @@
+            smallestBubble.topAnchor.constraint(equalTo: middleBubble.bottomAnchor, constant: Constants.SenderSmallBubbleConstraints.topAnchor),
+            smallestBubble.widthAnchor.constraint(equalToConstant: Constants.ChatBubbleConstraints.smallestBubbleWidth),
+            smallestBubble.heightAnchor.constraint(equalToConstant: Constants.ChatBubbleConstraints.smallestBubbleHeight)
+        ])
+    }
+    
+    private func setUpSenderMiddleBubbleConstraints() {
+        NSLayoutConstraint.activate([
+            middleBubble.trailingAnchor.constraint(equalTo: chatBubble.trailingAnchor, constant: Constants.SenderMiddleBubbleConstraints.trailing),
+            middleBubble.bottomAnchor.constraint(equalTo: chatBubble.bottomAnchor, constant: Constants.SenderMiddleBubbleConstraints.bottom),
+            middleBubble.widthAnchor.constraint(equalToConstant: Constants.ChatBubbleConstraints.middleBubbleWidth),
+            middleBubble.heightAnchor.constraint(equalToConstant: Constants.ChatBubbleConstraints.middleBubbleHeight)
+        ])
+    }
+    
+    private func setUpSenderChatBubbleConstraints() {
+        NSLayoutConstraint.activate([
+            chatBubble.topAnchor.constraint(equalTo: contentView.topAnchor, constant: Constants.SenderChatBubbleConstraints.top),
+            chatBubble.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: Constants.SenderChatBubbleConstraints.trailing),
+            chatBubble.bottomAnchor.constraint(equalTo: messageLabel.bottomAnchor, constant: Constants.SenderChatBubbleConstraints.bottom)
+        ])
+    }
+    
+    private func setUpRightMessageLabelConstraints() {
+        NSLayoutConstraint.activate([
+            messageLabel.topAnchor.constraint(equalTo: chatBubble.topAnchor, constant: Constants.SenderMessageLabelConstraints.top),
+            messageLabel.trailingAnchor.constraint(equalTo: chatBubble.trailingAnchor, constant: Constants.SenderMessageLabelConstraints.trailing),
+            messageLabel.bottomAnchor.constraint(equalTo: chatBubble.bottomAnchor, constant: Constants.SenderMessageLabelConstraints.bottom),
+            messageLabel.leadingAnchor.constraint(equalTo: chatBubble.leadingAnchor, constant: Constants.SenderMessageLabelConstraints.leading)
+        ])
+        messageLabel.textAlignment = .right
+    }
+    
+    private func setUpRightDateLabelConstraints() {
+        NSLayoutConstraint.activate([
+            dateLabel.topAnchor.constraint(equalTo: chatBubble.bottomAnchor, constant: Constants.SenderDateLabelConstraints.top),
+            dateLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: Constants.SenderDateLabelConstraints.trailing),
+            dateLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: Constants.SenderDateLabelConstraints.bottom),
+        ])
+    }
+    
+    func configure(with message: String, indexPath: IndexPath) {
```

Comment:
You should pass the model for presentation to your cell.
For instance:
```suggestion
    enum BubblePosition { case left, right }
    struct BubbleMessage { 
        let position: BubblePosition
        let message: String
    }
```